Sources/TuistCore/Graph/Mappers/GraphMapper.swift

```diff
@@ -6,6 +6,13 @@ public protocol GraphMapping {
     /// Given a graph, it maps it into another graph.
     /// - Parameter graph: Graph to be mapped.
     func map(graph: Graph) throws -> (Graph, [SideEffectDescriptor])
+    func map(graph: ValueGraph) throws -> (ValueGraph, [SideEffectDescriptor])
```

Comment:
I'd maybe add some documentation similar to above as this function is a core component of graph manipulation